File: 2st/Yeonwoo/홀짝에 따라 다른 값 반환하기.py
def solution(n):
    answer = 0
    if n%2==1:
        for i in range(1,n+1):
            if i%2==1:
                answer += i
    elif n%2==0:
        for i in range(1, n+1):
            if i%2==0:
                answer += i**2
    return answer

File: 4st/Seohyun/홀짝에 따라 다른 값 반환하기.js
function solution(n) {
  let answer = 0;

  if (n % 2 !== 0) {
    for (let i = n; n >= 0; n -= 2) {
      answer += n;
    }
    return answer;
  }

  if (n % 2 === 0) {
    for (let i = n; n >= 0; n -= 2) {
      answer += n * n;
    }
    return answer;
  }

  return answer;
}
